catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/dialog/OrientationDialogTest.java

```diff
@@ -0,0 +1,128 @@
+ */
+
+package org.catrobat.catroid.uiespresso.ui.dialog;
+
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.espresso.Espresso;
+import android.support.test.espresso.IdlingResource;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.ui.MainMenuActivity;
+import org.catrobat.catroid.ui.ProjectActivity;
+import org.catrobat.catroid.uiespresso.testsuites.Cat;
+import org.catrobat.catroid.uiespresso.testsuites.Level;
+import org.catrobat.catroid.uiespresso.util.UiTestUtils;
+import org.catrobat.catroid.uiespresso.util.rules.DontGenerateDefaultProjectActivityInstrumentationRule;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+import static junit.framework.Assert.assertTrue;
+
+import static org.catrobat.catroid.ui.SettingsActivity.SETTINGS_CAST_GLOBALLY_ENABLED;
+
+@RunWith(AndroidJUnit4.class)
+public class OrientationDialogTest {
+
+	@Rule
+	public DontGenerateDefaultProjectActivityInstrumentationRule<MainMenuActivity> baseActivityTestRule = new
+			DontGenerateDefaultProjectActivityInstrumentationRule<>(MainMenuActivity.class, true, false);
+
+	private IdlingResource idlingResource;
+	private String chromeCast = SETTINGS_CAST_GLOBALLY_ENABLED;
+	private boolean initialChromeCastSetting;
+
+	@Before
+	public void setUp() throws Exception {
+		SharedPreferences sharedPreferences = PreferenceManager
+				.getDefaultSharedPreferences(InstrumentationRegistry.getTargetContext());
+
+		SharedPreferences.Editor sharedPreferencesEditor = sharedPreferences.edit();
+
+		initialChromeCastSetting = sharedPreferences.getBoolean(chromeCast, false);
+
+		sharedPreferencesEditor.putBoolean(chromeCast, true);
+
+		sharedPreferencesEditor.commit();
+
+		baseActivityTestRule.launchActivity(null);
+
+		idlingResource = baseActivityTestRule.getActivity().getIdlingResource();
+		Espresso.registerIdlingResources(idlingResource);
+	}
+
+	@Test
+	//@category({Level.Smoke.class, Cat.AppUi.class, Cat.Gadgets.class})
+	@Category({Level.Smoke.class, Cat.AppUi.class, Cat.Gadgets.class})
+	public void newProject() {
+		onView(withId(R.id.main_menu_button_new))
+				.perform(click());
+
+		//check if dialog title is displayed
```

Comment:
please remove comments, not necessary here, were just for the smokeTest since it acts as an educational test